File: frontend/src/app/models/Student.ts
export class Student {
  id: number;
  name: string;
  cpf: string;
  teacherId: number;

  constructor(data: any) {
    this.id = data.id;
    this.name = data.name;
    this.cpf = data.cpf;
    this.teacherId = data.teacherId;
  }
}

File: backend/Commons/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Escola.Commons
{
    public class Student
    {
        public int? Id { get; set; }
        public string Name { get; set; }
        public string Cpf { get; set; }
        public int TeacherId { get; set; }
    }
}
